src/components/App/AppSagas.js

```diff
@@ -0,0 +1,33 @@
+import {call, put} from 'redux-saga/dist/redux-saga-effects-npm-proxy.cjs';
+
+import auth from 'auth';
+import {CHANGE_PAGE_ACTION} from 'components/App/AppActions';
+import {PAGES_ENUM} from 'components/App/AppConstants';
+import thoughtStorage from 'storage/thought-storage';
+import googleDriveApi from 'api/google-drive-api';
+import googleApiLoader from 'api/google-api-loader';
+import {hideSpinner, showSpinner} from 'components/Spinner/SpinnerSagas';
+
+function setPageAction(data) {
+  return put({type: CHANGE_PAGE_ACTION, data});
+}
+
+export function* loadApp() {
```

Comment:
I would assume that `loadApp` is where the app starts loading, makes first requests etc.

But in `rootSaga()` we call `signIn()`, not `loadApp()`. So are we beginning the app with `signIn()` instead of beginning it with `loadApp()`? I'm confused.

Reply:
That is naming issue. I will try to make it more clear.